Atari-Game-Player/pipelines/training_pipeline.py

```diff
@@ -0,0 +1,29 @@
+from zenml.pipelines import pipeline
+
+
+@pipeline
+def train_pipeline(game_wrap, build_dqn, replay_buffer, agent, get_information_meta, train):
+    """
+    It trains the agent.
+    
+    :param game_wrap: This is a function that returns a GameWrapper object. The GameWrapper object wraps
+    over the game that you want to train on. It has functions that can be used to get the available
+    actions, get the current state, etc
+    :param build_dqn: This is a function that returns a DQN. The parameters are the game_wrapper, which
+    is the game wrapper object that we created earlier
+    :param replay_buffer: The replay buffer is a class that holds all the experiences a DQN has seen,
+    and samples from it randomly to train the DQN
+    :param agent: The agent that will be used to play the game
+    :param get_information_meta: This is a function that returns the frame number, rewards, and loss
+    list
+    :param train: The function that will be called inside the train_pipeline function. This is the
+    function that will be called

```

Comment:
We usually use Google style docstrings with 
```
Args:
    game_wrap: ....
    
```

it might make sense to stay consistent across your code

Reply:
It's done